src/ceph-volume/plugin/zfs/Makefile

```diff
@@ -0,0 +1,88 @@
+.PHONY: clean clean-test clean-pyc clean-build docs help
```

Comment:
i'd suggest consolidate this `Makefile` with the `CMakeLists.txt` instead of having a self-contained mini project in Ceph.

Reply:
@tchaikov 
ATM it allows me to work on into without going thru the full cycle of cmake.
I can remove that from the PR.

But I do no really known how to build a CMakeLists.txt for a python project, 
other than that I copied the one from ceph-volume. 
I'd expect that to do the trick also for